From: Ralf Baechle <ralf@linux-mips.org>
Date: Sat, 23 Sep 2006 17:08:36 +0000 (+0100)
Subject: [MIPS] Have headers_install install <asm/cachectl.h> and <asm/sysmips.h>.
X-Git-Tag: linux-1.1.68
X-Git-Url: http://www.linux-mips.org/git?p=linux.git;a=commitdiff;h=e70d9b79a35ac822ec5913278d2472aa728d1476

[MIPS] Have headers_install install <asm/cachectl.h> and <asm/sysmips.h>.

Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
(cherry picked from 2ff556ab120549346660fd7e0cd25782ebe5381d commit)
---

--- a/include/asm-mips/Kbuild
+++ b/include/asm-mips/Kbuild
@@ -1,3 +1,3 @@
 include include/asm-generic/Kbuild.asm
 
-header-y += sgidefs.h
+header-y += cachectl.h sgidefs.h sysmips.h
